app/views/previews/show.html.erb

```diff
@@ -10,22 +10,21 @@
         <%= @dataset.title %>
         <span class="heading-secondary dgu-home-spacer__bottom"><%= @datafile.name %></span>
       </h1>
+
       <% if @datafile.csv? && @datafile.preview.exists? %>
         <p>You're previewing the first <%= @datafile.preview.line_count %> rows of this file.</p>
-      <% end %>
-      <%= link_to t('.download_file'),
-      @datafile.url,
-        class: 'button',
-        :data => {
-          :'ga-event' => 'preview',
-          :'ga-format' => (@datafile.format.presence || 'n/a').upcase,
-          :'ga-publisher' => @dataset.organisation.name
-        }
+        <%= link_to t('.download_file'),
+        @datafile.url,
+          class: 'button',
+          :data => {
+            :'ga-event' => 'preview',
```

Comment:
Should we be sending a 'preview' event for a download?

Reply:
@nicholsj is it possible to check GA and see whether this is doing the right thing?  Not sure what events we're expecting to store in GA.

Reply:
Desired behaviour: 

Download : Category = datafile, action = download, label= <datafile url>

Preview :  Category = datafile-preview, action = click, label= <datafile url>
https://trello.com/c/WLA2YUze/328-add-ga-event-tracking-to-dataset-preview-links-s

What I'm seeing in GA:

Clicking the preview does send an event as desired. 
But clicking the download button from the preview page sends a preview event not a download one.

Reply:
@nicholsj 
cc @rossjones 

There is a download event triggered by clicking download on the datafile table (on the dataset show page). How do you want to differentiate the download from the preview page? Are you happy with `Datafile-preview, action = download`

Reply:
[Trello card #385](https://trello.com/c/ih1BdKOC/385-add-ga-datafile-download-event-on-preview-page)